### ISSUE TITLE
I want to use vite-stripper to strip a backend only package

### ISSUE THREAD

**Issue by noam-honig** [jycouet/kitql#459]
**Is your feature request related to a problem? Please describe.**

I use mongodb's ObjectId class when I save to the db - it's part of mongodb package that is obviously server only - I would like to strip that package

**Describe the solution you'd like**

allow me to declare packages to strip

**Describe alternatives you've considered**

Tried the build & external vite config options, but in this case, the import is at the top of the file - so vite still tries to load it - I don't want that

**Additional context**

See the following codesandbox - as soon as I add frontend code that uses that entity, I get an error about mongodb

https://codesandbox.io/p/devbox/mong-db-remult-pch8zc?file=%2Fsrc%2Findex.ts%3A5%2C1-15%2C2

**Comment by jycouet** [comment]
Here it's an easy case because it's not used in the code.
Can you show me an example we it's used? And what should I do with it?

**Comment by noam-honig** [comment]
Here's a working example where I use the `ObjectId` type from mongo db (required for storing it in the db) but it can't run in the front end

https://codesandbox.io/p/devbox/hungry-blackwell-xjzwyy

**Comment by jycouet** [comment]
I'm wondering if I can just comment for the browser bundle?
```ts
@Fields.string({
    valueConverter: {
      fromDb: (x) => x?.toString(),
      toDb: (x) => {
        // const r = new ObjectId(x)
        console.log(r)
        return r
      }
    }
  })
```

Or "they" are too smart and will say that r is not defined... I'll check it out.

Thx for the example

**Comment by noam-honig** [comment]
Since this code will never execute you can replace it with null or throw an exception

**Comment by jycouet** [comment]
Fixed in `0.3.1`, confirmed with: https://codesandbox.io/p/devbox/packages-striper-lfq6rf?file=%2Fvite.config.ts%3A10%2C28
